pkgs/by-name/ou/outputcheck/package.nix

```diff
@@ -0,0 +1,33 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "outputcheck";
+  version = "0.4.2";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "stp";
+    repo = "OutputCheck";
+    rev = "refs/tags/${version}";
+    hash = "sha256-QAlX+dmDKawnEzxtn9TToEmsA9jjzwO04l4bLM3fR/g=";
+  };
+
+  # Fix deprecated 'U' mode in python 3.11
+  # https://github.com/python/cpython/blob/3.11/Doc/library/functions.rst?plain=1#L1386
+  postPatch = ''
+    substituteInPlace OutputCheck/Driver.py \
+      --replace "argparse.FileType('rU')" "argparse.FileType('r')"
```

Comment:
I'm not sure if upstream will respond, but how about submitting an issue or PR?

Reply:
Hmm, will omit this first on init. Then submit an issue and bug fix PR.

Reply:
Okay will return it back then.